src/openforms/config/checks.py

```diff
@@ -119,7 +121,7 @@ def get_plugin_entry(self, plugin: AbstractBasePlugin | ConfigCheckable) -> Entr
         try:
             actions = plugin.get_config_actions()
         except Exception as e:
-            actions = [
+            actions: list[tuple[str, str]] = [
```

Comment:
```suggestion
            actions: list[Action] = [
```
there's a type alias for this already